command/build.go

```diff
@@ -360,6 +360,26 @@ func maxAppNameLen(apps []*baur.App) int {
 	return maxLen
 }
 
+func appsWithBuildCommand(apps []*baur.App) []*baur.App {
+	res := make([]*baur.App, 0, len(apps))
+
+	maxAppNameLen := maxAppNameLen(apps) + 4
```

Comment:
- shadowing the package function with a local variable;
- What's the `+ 4`? It must be a const or at least have a relevant comment.

Reply:
I improved both :-)